bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/handler/VehicleHandler.java

```diff
@@ -198,336 +165,735 @@ public void initialize() {
-            return;
-        } else if (!online) { // only update if thing isn't already ONLINE
-            updateStatus(ThingStatus.ONLINE);
-        }
-
-        // Mileage for all cars
-        String odoUrl = String.format(ODO_URL, config.get().vin);
-        if (accountConfigAvailable()) {
-            if (accountHandler.get().config.get().odoScope) {
-                call(odoUrl);
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        /**
+         * Commands shall be not that frequent so trace level for identifying problems should be feasible
+         */
+        logger.trace("Received command {} {} for {}", command.getClass(), command, channelUID);
+        if (command instanceof RefreshType) {
+            if (MB_KEY_FEATURE_CAPABILITIES.equals(channelUID.getIdWithoutGroup())
+                    || MB_KEY_COMMAND_CAPABILITIES.equals(channelUID.getIdWithoutGroup())) {
+                accountHandler.ifPresent(ah -> {
+                    ah.getVehicleCapabilities(config.get().vin);
+                });
             } else {
-                logger.trace("{} Odo scope not activated", this.getThing().getLabel());
+                // deliver from event storage
+                ChannelStateMap csm = eventStorage.get(channelUID.getId());
+                if (csm != null) {
+                    updateChannel(csm);
+                }
             }
-        } else {
-            logger.trace("{} Account not properly configured", this.getThing().getLabel());
-        }
-
-        // Electric status for hybrid and electric
-        if (uid.equals(BEV) || uid.equals(HYBRID)) {
-            String evUrl = String.format(EV_URL, config.get().vin);
-            if (accountConfigAvailable()) {
-                if (accountHandler.get().config.get().evScope) {
-                    call(evUrl);
+            // ensure unit update
+            unitStorage.remove(channelUID.getIdWithoutGroup());
+        } else if (Constants.GROUP_VEHICLE.equals(channelUID.getGroupId())) {
+            /**
+             * Commands for Vehicle
+             */
+            if (OH_CHANNEL_IGNITION.equals(channelUID.getIdWithoutGroup())) {
+                String supported = thing.getProperties().get(MB_KEY_COMMAND_ENGINE_START);
+                if (Boolean.FALSE.toString().equals(supported)) {
+                    logger.trace("Engine Start/Stop not supported");
+                } else {
+                    int commandValue = ((DecimalType) command).intValue();
+                    if (commandValue == 4) {
+                        String pin = accountHandler.get().config.get().pin;
+                        if (Constants.NOT_SET.equals(pin)) {
+                            logger.trace("Security PIN missing in Account bridge");
+                        } else {
+                            EngineStart eStart = EngineStart.newBuilder().setPin(supported).build();
```

Comment:
```suggestion
                            EngineStart eStart = EngineStart.newBuilder().setPin(pin).build();
```

I guess this is a bug.

Reply:
Sure, my bad!